src/components/Pagination/Pagination.jsx

```diff
@@ -184,12 +182,12 @@ class Pagination extends Component {
 
       return (
         <a
+          aria-current={this.props.page === pageNumber ? 'true' : null}
```

Comment:
Can we get a test to check that `aria-current` is set when the `page` prop matches the `pageNumber`?

Reply:
Hey @bkjohnson I just added that test.